frontend/src/app/application-forms/temporary-outfitters/temporary-outfitters.component.ts

```diff
@@ -182,13 +189,118 @@ export class TemporaryOutfittersComponent implements DoCheck {
   }
 
   numberOfFilesToUpload() {
-    this.numberOfFiles = this.applicationFieldsService.parseNumberOfFilesToUpload([
-      this.applicationForm.get('applicantInfo.goodStandingEvidence'),
-      this.applicationForm.controls.guideIdentification,
-      this.applicationForm.controls.operatingPlan,
-      this.applicationForm.controls.liabilityInsurance,
-      this.applicationForm.controls.acknowledgementOfRisk
-    ]);
+    this.numberOfFiles = this.applicationFieldsService.getNumberOfFiles();
+  }
+
+  removeUnusedData() {
```

Comment:
could we add some type doc documentation.

Reply:
added